web/template/course-card.gohtml

```diff
@@ -1,21 +1,29 @@
 {{define "course-card"}}{{- /*gotype: TUM-Live/model.Course*/ -}}
 <div class="pt-1 group course{{.Model.ID}}">
     <div class="flex items-center">
-        <i class="fancyeye opacity-0 group-hover:opacity-100 transition-colors duration-200 dark:hover:text-white hover:text-black text-gray-500 mr-2 cursor-pointer"
+        <i class="fancyeye opacity-0 group-hover:opacity-100 transition-colors duration-200 dark:hover:text-whitehover:text-black text-gray-500 mr-2 cursor-pointer"
            title="Hide this course" onclick="hideCourse({{.Model.ID}}, {{.Name}})"></i>
         <a href="/course/{{.Year}}/{{.TeachingTerm}}/{{.Slug}}" class="text-lg text-3">{{.Name}}</a>
     </div>
     <div class="flex">
         <i class="icon-eye invisible mr-2"></i>
         <p class="text-5 font-sans font-light text-sm">
+            {{/*Order of conditions is important*/}}
             {{if .IsLive}}
                 Live now.
-            {{else}}
-                {{if .HasNextLecture}}
-                    {{$next := .GetNextLectureDate}}Next Lecture: {{printf "%v %02d. %02d:%02d" $next.Month $next.Day $next.Hour $next.Minute}}
+            {{else if .HasNextLecture}}
+                {{if .IsNextLectureStartingSoon}}
+                    <div class="timer text-5 font-sans font-light text-sm" x-data="timer('{{.GetNextLectureDateFormatted}}', '{{.GetNextLecture.IsRegularStream}}', false)" x-init="init();">
+                        In&nbsp;<span x-text="time().minutes"></span>&nbsp;Minutes •&nbsp;
```

Comment:
If the stream should already have started but didn't this should display something like "starting soon" instead of "in -10 minutes".
Also if there's only one minute left it would be nice if this was in one minut**e** (no plural)

Reply:
Oh yeah, forgot about the singular case :see_no_evil: I thought it could be useful to see the overdue time, but I agree that its pretty ugly this way, I will fix that.